tools/install-tcmalloc.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+#
+# Script to install libtcmalloc.so in the current WiredTiger git
+# repository.
+#
+# This is serves the same purpose, but is NOT THE SAME as:
+#
+# test/evergreen/tcmalloc_install_or_build.sh
+#
+# and the tcmalloc source version referenced here should be updated in
+# sync with that script.
+
+set -euf -o pipefail
+
+HERE=$(git rev-parse --show-toplevel) || echo "FATAL Run in root of git workspace"
```

Comment:
`set -e` doesn't have an effect on this.
If `git rev-parse` fails, then the exit status of the whole command will be the `echo`'s one which is always "success", therefore the script will continue execution even if `rev-parse` fails and the error is reported.

A good practice is to add this function:
```bash
die() { echo "$@"; exit 1; }
```

and use it instead of `echo` like this:
```bash
command || die "error"
```

Same applies to other error checks.

Reply:
Done